Resources/Prototypes/Nyanotrasen/Entities/Structures/Machines/vending_machines.yml

```diff
@@ -72,3 +72,23 @@
   - type: WiresVisuals
   - type: AccessReader
     access: [["Service"]]
+type: entity
+  parent: VendingMachine
+  id: VendingMachineBoxingDrobe
+  name: Boxing Drobe
+  description: All you need to be a boxing station star.
+  components:
+  - type: VendingMachine
+    pack: BoxingDrobeInventory
+  - type: Sprite
+    sprite: Nyanotrasen/Structures/Machines/VendingMachines/boxingdrobe.rsi
+    layers:
+    - state: "off"
+      map: ["enum.VendingMachineVisualLayers.Base"]
+    - state: "off"
```

Comment:
the unshaded state in the RSI should be normal-unshaded, right does this work in game?

Reply:
I haven't tested in visual code as I can't seem to get it working